pylib/cqlshlib/copyutil.py

```diff
@@ -884,7 +884,7 @@ def __init__(self, fname, options):
         self.max_rows = options.copy['maxrows']
         self.skip_rows = options.copy['skiprows']
         self.fname = fname
-        self.sources = None  # must be created later due to pickle problems on Windows
+        self.sources = None
```

Comment:
Without Windows' issues, should we instead initialize this at declaration time now?

Reply:
looking into this, this one seems to be easy to move to constructor.